data-access-analysis-tools/portal-intro.rst

```diff
@@ -394,3 +394,79 @@ The color-composite background shows the relevant DC2 simulated sky region.
 
 **Learn More.**
 See also :ref:`DP0-2-Tutorials-Portal` for a tutorial using additional image types and more of the Portal's image-related functionality.
+
+.. _Portal-Intro-URL-APIs:
+
+Public URL API for the Portal
```

Comment:
Please spell out acronyms the first time they are used (URL, API) in this document.